manuscript/revision/response_to_editor.tex

```diff
@@ -0,0 +1,225 @@
+limits of the journal allowed for it.}
+
+My main comment is that tsbrowse does not actually seem to be an ARG visualisation tool. It is a useful way 
+to visualise mutations, tree-sequence edges, and summary statistics, but nowhere will you actually see a tree 
+(much less an ARG). On rereading the paper I see that the manuscript has been carefully worded not to include 
+such a claim, but a casual reader might expect tsbrowse to be able to do so, as other tools such as IcyTree can. 
+I do get fed up with reviewers tossing additional requests on top of an already polished piece of work, but 
+in this case I think it would be a helpful addition! - would it really be too much work for tsbrowse to be 
+able to draw at least the local trees? (For researchers with smaller datasets, being able to interrogate the 
+fine details of an ARG would be equally valuable as the complementary selling-points around scalability.) At 
+the very least I would like to see the manuscript at least be a little more up-front about this omission.\\
+
+\textcolor{blue}{We thank the reviewer for making this important observation, and agree that drawing 
+(parts of) local trees would be a great feature. For smaller datasets, tools such as arg-visualiser 
+(\url{https://pypi.org/project/tskit-arg-visualizer/}) and existing tskit infrastructure, for example 
+the \texttt{draw\_text()} and \texttt{draw\_svg()} already provide this functionality. A key 
+motivation for this work is to guide large-scale inference, and implementing tree visualisation that is interpretable 
+at this scale would require significant thought towards design as well as considerable effort. We therefore 
+think that implementing this feature is not feasible for this work and is better scoped as future work or as 
+a separate endeavour. In the text, we do state that tsbrowse is a visualiser for the information structure 
+in ARGs, which we hope implies that it is a visualiser for the properties of the ARG, rather than the ARG 
+topology itself. We have also already explicitly stated in the Discussion that drawing local trees is an 
+important future direction.}\\
+
+Minor comments:\\
+
+Manuscript p1: 'it is only with recent breakthroughs in inference methods...that practical application 
+has become possible'. I appreciate the need to emphasise the recent increase in interest in inference for ARGs, 
+but I think this is selling older methods a little short. There are ARG inference methods going back at least 
+as far as Griffiths and Marjoram (1996) 'Ancestral Inference from Samples of DNA Sequences' JCB 3(4):479-502. 
+True they are not scalable in the manner of recent methods, but they *are* practical applications, and they have 
+the advantage that they actually sample from the claimed model without introducing any model heuristics.
+\textcolor{blue}{Thanks to the reviewer for their feedback on this statement on early methods. We agree, 
```

Comment:
Thank you for pointing this out, and you are right that we are somewhat overstating the case for recent methods. We have changed the word "practical" to "widespread", which we feel is an uncontroversial statement. The history of ARG inference is a complicated and confusing subject, dealt with exhaustively by Wong et al 2022, and would be impossible to accurately summarise in the available space. We note, though, that two of the recent methods cited here (ARGweaver and SINGER) sample ARGs from the posterior defined by a coalescent model in a statistically rigorous manner.